### DISCUSSION TITLE
"Getting Started guide" - Hyperlink leads to a 404

### DISCUSSION THREAD

**Discussion by adrns012** [community/community#6628] [Projects and Issues]
Hi everyone, I just got access to copilot and I'm super excited to get going, however, I've encountered a slight snag. Generally speaking, I like to refer to documentation first and I'm unable to at this time. 

As the title suggests, clicking the "Getting Started guide" hyperlink leads me to a 404 page. Is this page temporarily down? 

Thank you in advance! 

Kindly,
Adrian

**Comment by cwinters8** [comment]
The getting started guide link is to a private repo - my guess is someone has to add each of us to it.

**Reply by adrns012** [reply]
Oh okay, thanks for the prompt reply Clark! Sounds good.
